HelpSource/Classes/Float.schelp

```diff
@@ -65,3 +90,67 @@ pi.asStringPrec(6)
 7.4.asStringPrec(5)
 7.4.asStringPrec(50)
 ::
+
+subsection::Using Floats as replacement for Integers
+
+In SuperCollider, Floats are 64-bit wide. Because  an link::Classes/Integer:: is only 32-bit, it can capture only whole numbers up to code::-2147483648 .. +2147483647::, which is code::2^31:: (one bit is for the sign), or about code::2 x 10^9::.
+
+Therefore, in some situations it can be useful to calculate with floats also when only whole numbers are needed. You can use 64-bit floats for integer calculations up to code::± 9007199254740992:: (code::2^53::, or about code::9 x 10^15::). Note that you still may get floating point errors when you apply certain operations.
+
+
+code::
+// compare factorial:
+f = { |x| if(x < 2) { x } { x * f.(x - 1) } };
+f.(14); // integer
+f.(18) // integer overflow: -898433024
+f.(18.0) // float is ok.
+// check the ratio between adjacent factorials:
+f.(18.0) / f.(17.0) == 18 // true
+// 18 is already the largest possible factorial representable in 64-bit float (< 2^53)
+{:x, x<-(1.0..40), f.(x) < (2 ** 53) }.all.last
+// but the correctness for comparing adjacent factorials goes up much higher (to 170)
+{:x, x<-(1.0..180), f.(x) / f.(x - 1) == x }.all.last
+f.(170.0) // 7.257415615308e+306.
+f.(170.0) / f.(169.0) // => 170.0
+::
```

Comment:
I think you should cut down on the examples, possibly by removing this one (c.f. #3345)

Reply:
I've restructured the whole a bit, I hope it is better now.